src/Endatix.Core/Handlers/SubmissionCompletedHandler.cs

```diff
@@ -1,13 +1,10 @@
-using System.Threading;
-using System.Threading.Tasks;
 using Microsoft.Extensions.Logging;
-using Endatix.Core.Features.Email;
 using Endatix.Core.Events;
 using MediatR;
 
 namespace Endatix.Core.Handlers;
 
-internal sealed class SubmissionCompletedHandler(IEmailSender emailSender, ILogger<SubmissionCompletedHandler> logger) : INotificationHandler<SubmissionCompletedEvent>
+internal sealed class SubmissionCompletedHandler(ILogger<SubmissionCompletedHandler> logger) : INotificationHandler<SubmissionCompletedEvent>
```

Comment:
Are this change and the one in src/Endatix.Infrastructure/Email/SendGridEmailSender.cs part of the solution of this issue or they fix something else?

Reply:
This one dependency was forcing initialization of the SendGrid connector requiring developers to add the required SendGrid API Key, so it was a bit unnecessary, at least to say as it complicates initial vanilla setup. One of the other issues that were outdated in the sample.